big_tests/run_common_test.erl

```diff
@@ -420,6 +443,8 @@ group_by(F, L) ->
 host_cluster(Host) -> host_param(cluster, Host).
 host_node(Host)    -> host_param(node, Host).
 host_vars(Host)    -> host_param(vars, Host).
+%% Directory name in _build
+rel_name(Host)     -> host_param(relname, Host).
```

Comment:
Perhaps we could simply rename keys in `hosts` so additional param won't be necessary at all?

Reply:
I was thinking about it.

We can't really.
Because relname is not nodename.

Rebar allows to build a mix of presets, so the name would be something like "preset1+preset2".

I would rather keep relname as a separate entity from any other key, we currently have in hosts (i.e. node, vars, cluster, domain and some more).

Reply:
also, in DEV_NODES we specify relnames.
and the whole meaning of DEV_NODES is "enable preset on a node in the release directory. Use hosts key in tests.config to figure out which node is which".

BTW, cool feature in presets. We can actually build releases like "mim1+with_all", if we create with_all preset.

Reply:
I will think, what changes are required to be able to specify HOSTS in DEV_NAMES, instead of relnames.

Reply:
OK, I understand. Indeed release names could change and it would be pretty inconvenient to change these names in all tests that use node names. :)

Reply:
Ah, we already use DEV_NODES in `tools/start-nodes.sh` and it wants build names.
There is no easy way to access test.config from bash.
That was the reasoning behind adding an extra key.

Reply:
And it's not easy to write testhostname_to_relname and relname_to_testhostname functions without explicitly defining mapping. 

So, I would rename fed test host into fed1 and reg into reg1.